internal/cmd/test/test.go

```diff
@@ -48,7 +41,7 @@ var testCmd = &cobra.Command{
 			}
 
 			// Read and remove --force-migrations flag from args:
-			forceMigrations = strings.Contains(strings.Join(args, ""), "--force-migrations")
+			var forceMigrations = strings.Contains(strings.Join(args, ""), "--force-migrations")
```

Comment:
in other places the `flag` package was used for things like this, should we change that here too?

Reply:
The test command is a bit different than the rest since it passes flags to the go test command. We can take this topic for another PR.

Reply:
I see, yes then lets have this postponed for a different PR.